practice/react-practice-one/src/components/SearchBar/SearchBar.jsx

```diff
@@ -0,0 +1,16 @@
+import SearchIcon from '../Icons/SearchIcon';
+import InputField from '../InputFields/InputField';
+
+/* Import CSS */
+import './searchBar.css';
+
+const SearchBar = () => {
+  return (
+    <div className="search-container input-form">
+      <SearchIcon />
+      <InputField type="search" placeholder="Search for food, coffee, etc.." />
```

Comment:
@JennyDoJD similar to other component, how about if we using this component in other place and modify the placehoder